projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -102,6 +104,12 @@ export class TuiLineClampComponent implements AfterViewInit {
         this.initialized = true;
     }
 
+    ngAfterViewChecked(): void {
```

Comment:
could you add e2e tests, please, for that case

Reply:
Unfortunately, I didn’t write e2e tests and I’m not sure what to do quickly and efficiently (

Reply:
Ok. I will try